db/beer.go

```diff
@@ -0,0 +1,19 @@
+package db
+
+//Beer is an entity that represents a record in the database
+type Beer struct {
```

Comment:
I'll expand the idea of the Beer Package here.

Beer should have it's own package, it could have it's own "New(...)" func... so that:

- We have everything related to the Beer in one place
- We can Unit Test the Beer decoupled from the rest of the packages
- We can use the Beer in a more effective way